bin/git-utimes

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Change files modification time to their last commit date
+#
+# Copyright (c) 2020 Vitaly Chikunov <vt@altlinux.org>.
+# SPDX-License-Identifier: MIT
```

Comment:
Please don't add the Copyright and License. We already have and only have a LICENSE file.